examples/ZeroUserCodeLightup.Net6/dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build-env
+WORKDIR /app
+
+# Copy everything else and build
+COPY . ./
+# Adding a reference to hosting startup package
+# Remove '--prerelease' if you want to stay only on stable version.
+RUN dotnet add package Microsoft.ApplicationInsights.Kubernetes.HostingStartup --prerelease
+RUN dotnet publish -c Release -o out
+
+# Build runtime image
+FROM mcr.microsoft.com/dotnet/aspnet:6.0
+
+# Create an argument to allow docker builder to passing in application insights key.
```

Comment:
Let's not do that. Instead, explain to the user how to ensure that `APPINSIGHTS_INSTRUMENTATIONKEY` is set when running the image locally using Docker, as well as when running on Kubernetes.

Reply:
Done.